modules/sdk-coin-core/src/lib/constants.ts

```diff
@@ -0,0 +1,14 @@
+export const mainnetValidDenoms = ['ucore'];
+export const mainnetAccountAddressRegex = /^(core)1(['qpzry9x8gf2tvdw0s3jn54khce6mua7l']{38})$/;
+export const mainnetValidatorAddressRegex = /^(corevaloper)1(['qpzry9x8gf2tvdw0s3jn54khce6mua7l']{38})$/;
+export const mainnetContractAddressRegex = /^(core)1(['qpzry9x8gf2tvdw0s3jn54khce6mua7l']+)$/;
+export const MAINNET_ADDRESS_PREFIX = 'core';
+
+export const testnetValidDenoms = ['utestcore'];
+export const testnetAccountAddressRegex = /^(testcore)1(['qpzry9x8gf2tvdw0s3jn54khce6mua7l']{38})$/;
+export const testnetValidatorAddressRegex = /^(testcorevaloper)1(['qpzry9x8gf2tvdw0s3jn54khce6mua7l']{38})$/;
+export const testnetContractAddressRegex = /^(testcore)1(['qpzry9x8gf2tvdw0s3jn54khce6mua7l']+)$/;
```

Comment:
instead of having different regex for testnet and mainnet, we can combine them as well.
Check sdk-coin-hash for this, like -
`export const accountAddressRegex = /^(tp|pb)1(['qpzry9x8gf2tvdw0s3jn54khce6mua7l']{38})$/;`

Reply:
I've implemented the utils to check the account & denomination formats based on the NetworkType thus I've kept it separate for testnet & mainnet

Reply:
ya, both way it works for me.